core/Cargo.toml

```diff
@@ -17,6 +17,7 @@ aes = "0.6"
 base64 = "0.13"
 byteorder = "1.4"
 bytes = "1.0"
+flate2 = "1.0"
```

Comment:
Could be removed again if we decide not to support gzip.

Reply:
While I think `gzip` responses are an overoptimization on the part of Spotify, we might as well be a good netizen and support it. I don't think binary sizes will matter much, so it's more a question of dependency stability and maintainability.

Reply:
We could also use [`async-compression`](https://docs.rs/async-compression/0.3.8/async_compression/), which is the same as `reqwest` uses.

Reply:
Your call, I am not partial to either. Doesn't `async-compression` in turn depend on `flate2`?